app/src/main/res/layout/fragment_posts_list.xml

```diff
@@ -9,12 +10,129 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
+        <LinearLayout
+            android:id="@+id/linearLayout"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent">
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="48dp">
```

Comment:
joł te wszytskie wartości jak 48 dp to wyrzuć do pliku dimens.xml proszę

Reply:
Wszystko przerzucę :)

Reply:
Wrzucone